webui/react/src/components/Table.tsx

```diff
@@ -132,7 +133,7 @@ export const taskNameRenderer: TaskRenderer = (id, record) => (
   <div>
     <ConditionalWrapper
       condition={canBeOpened(record)}
-      wrapper={(ch) => <Link path={paths.interactive(record)}>{ch}</Link>}>
+      wrapper={(ch) => <a onClick={() => openBlank(paths.interactive(record))}>{ch}</a>}>
```

Comment:
`<Link>` component has `external` prop but even if its true, it doesnt open new tab, so I chose `<a>` instead